src/tests/readytorun/multifolder/multifolder.csproj

```diff
@@ -40,6 +40,10 @@ $(CLRTestBatchPreCommands)
     echo %scriptPath%\..\FolderA\FolderA\FolderA.dll>>%__ResponseFile%
     echo %scriptPath%\..\FolderB\FolderB\FolderB.dll>>%__ResponseFile%
 
+    REM Suppress GC stress mode for the duration of Crossgen2 execution
```

Comment:
Shall we say standalone GC instead of GC stress mode?

Reply:
yeah will update